config/setup

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+full_path=$(realpath $0)
+dir_path=$(dirname $full_path)
+
+# The google-java-format version must match the one used in the pom.xml file
+version="1.24.0"
```

Comment:
you might also be able to read the version from the pom using mvn help:evaluate
VERSION=$(mvn help:evaluate -Dexpression=google-java-format.version -q -DforceStdout)

Reply:
use version form pom.xml